common/version.c

```diff
@@ -54,6 +54,11 @@ eprint_version(void)
 #else
     const char *has_dbus = "✘";
 #endif
+#ifdef WITH_XCB_ERRORS
+    const char *has_xcb_errors = "✔";
+#else
+    const char *has_xcb_errors = "✘";
```

Comment:
btw that's obviously out of scope of this PR but since i noticed this line here it reminded me a thing:

on some systems i've noticed the font didn't had those characters, so it was just a square or empty space

and sometimes they're just a bit shifted in position (if font doesn't have them and using from fallback font which have incompatible dimensions of characters)

how do you think is it worth creating an issue to discuss/address it or it could just waste time on bikeshedding?

Reply:
Well, if you have a suggestion for what to do instead, try a PR. That would avoid bikeshedding. I do not have any ideas right now.

Reply:
https://github.com/awesomeWM/awesome/pull/2687/files